camayoc/tests/qcs/api/v1/utils.py

```diff
@@ -22,14 +22,37 @@ def wait_until_state(scan, timeout=120, state='completed'):
 
     The default state is 'completed'.
 
+    Valid options for 'state': 'completed', 'failed', 'paused',
+    'canceled', 'running', 'stopped'.
+
+    If 'stopped' is selected, then any state other than 'running' will
+    cause `wait_until_state` to return.
+
     This method should not be called on scan jobs that have not yet been
     created, are paused, or are canceled.
 
     The default timeout is set to 120 seconds, but can be overridden if it is
     anticipated that a scan may take longer to complete.
     """
+    TERMINAL_STATES = ['completed', 'failed', 'paused', 'canceled']
+    if state not in ['completed',
```

Comment:
`if state not in ['running', 'stopped'] + TERMINAL_STATES:` ?

Reply:
Also I think CLI can benefit of the scan status if we make them a constant. We could create two constants: SCAN_TERMINAL_STATUSES and SCAN_STATUSES.